### PR TITLE
Collapse: Change icon

### PR DESCRIPTION
**Pull request by andresmgot**
**What this PR does / why we need it**:

In all the other places I could find (e.g. CollapsableSection), the `angle-down` icon is rendered when the item is expanded (and we use the `angle-right` when the item is closed). This PR adapts the `Collapse` component to that behavior:

Peek 2022-03-29 12-01

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-ui/src/components/Collapse/Collapse.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import React, { FunctionComponent, useState } from 'react';
 import { css, cx } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import React, { FunctionComponent, useState } from 'react';
 
 import { useStyles2 } from '../../themes/ThemeContext';
 import { Icon } from '../Icon/Icon';
```
Comment (andresmgot) [inline packages/grafana-ui/src/components/Collapse/Collapse.tsx:6]: this was automatically ordered

**Comment by Clarity-89** [comment]
Is it just me or is that icon not vertically aligned with the text? 🤔

**Comment by andresmgot** [comment]
> Is it just me or is that icon not vertically aligned with the text? thinking

right, is a bit misplaced (it was like that before). Happy to move it but is there a preferred way of doing that?

**Comment by torkelo** [comment]
Yea I the misalignment of the icon and text is really triggering to me :) 

Just make the padding for the header symmetrical   (it's 8px up and 4px down right now, making it 8px up and 8px down fixes the issue) 

Screenshot from 2022-03-29 08-19-48

**Comment by andresmgot** [comment]
done!

**Review by Clarity-89 — APPROVED** [review]

**Pull request merged**